defaults/main.yml

```diff
@@ -23,6 +23,10 @@ __slurm_user_name: "{{ (slurm_user | default({})).name | default('slurm') }}"
 # TODO: this could be incorrect, use the group collection from galaxyproject.galaxy
 __slurm_group_name: "{{ (slurm_user | default({})).group | default(omit) }}"
 
+
+# install from custom debian repos
+__slurm_configure_repos: false
```

Comment:
From what I see in this file, the prefix `__` seems to be used for dynamic variables. I guess this new variable should be `slurm_configure_repos` instead of `__slurm_configure_repos`. (note that this is the variable `slurm_configure_repos` which  you are usinng in the tasks)

Reply:
it's a typo i use the forme slurm_configure_repos in all my tasks it's boolean variable not dynamique

Reply:
fixed